src/main/java/org/testng/internal/PackageUtils.java

```diff
@@ -31,7 +32,7 @@
   private static String[] testClassPaths;
 
   /** The additional class loaders to find classes in. */
-  private static final List<ClassLoader> classLoaders = new Vector<>();
+  private static final Collection<ClassLoader> classLoaders = new ConcurrentLinkedDeque<>();
```

Comment:
I am not sure about the intent behind having this collection as a concurrent one. The method that directly adds elements into this collection is `org.testng.internal.PackageUtils#addClassLoader` and I don't see any direct invocations to this method from within TestNG. But that being said, I don't know if there are external customers (like the IntelliJ plugin or the surefire plugin or the eclipse plugin) which makes use of this method and how do they do it. If this method gets invoked from concurrent threads then there's a very good chance of triggering `ConcurrentModificationException` (if an `ArrayList` is used). That was why I was playing it safe by making use of a concurrent collection (In short adhering to the existing codebase, but just replacing `Vector` with a concurrent collection.